testeng/jobs/upgradePythonRequirements.groovy

```diff
@@ -16,10 +16,14 @@ package testeng
 // Cron value for once per week, sometime during: midnight-7:59am ET, Monday-Friday (times are in UTC)
 // See https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/hudson/triggers/TimerTrigger/help-spec.jelly
 def cronOffHoursBusinessWeekday = 'H H(4-11) * * H(1-5)'
+// Cron value for once per week, sometime during: 5am-12:59pm Pakistan Time, Monday-Friday (times are in UTC)
+def cronOffHoursBusinessWeekdayLahore = 'H H(0-7) * * H(1-5)'
```

Comment:
I personally liked when these would appear as new work in the morning. Are most people on arbi-bom not starting until after 1pm your time? If that is your adjusted schedule, this is fine. Otherwise, you can bump that up a couple of hours. I found it nicer than having it appear as an interruption in the middle of the day.
In any case, this schedule should be all about your convenience. So, whatever you think best is fine.

Reply:
Yes, you are right, most of us usually start work around 12-1 PM and these hours are agreed upon by the team.

Reply:
If so, you could switch to H(0-5) [or H(0-6)] from H(0-7) so you have the PRs waiting when you start. But, again, totally your decision.

Reply:
Yes, that sounds good, would that be fine, If I reduce the window to 5 hours?

Reply:
Changed as suggested.